<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_username_2">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>260</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_password_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailLineEdit_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_email">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>360</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="roleLineEdit_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_role">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_role_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="roleLineEdit_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>410</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>70</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QListWidget" name="productListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="descriptionLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="priceLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="imageLineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="publishButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Publish</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
